packages/account/src/Styles/account.scss

```diff
@@ -442,6 +442,21 @@ $MIN_HEIGHT_FLOATING: calc(
             border-top: 1px solid var(--general-active);
         }
 
+        &__authentication-message {
+            display: flex;
+            flex-direction: row;
+            align-items: flex-start;
+            padding: 0.8rem;
+            gap: 0.8rem;
+            height: fit-content;
+            background: #377cfc29;
```

Comment:
can we check a better way for this? i think this wont apply in dark mode

Reply:
Asking product design team to give me a color tone for dark mode 🙏